<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeThreshold</class>
 <widget class="QDialog" name="ChargeThreshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Battery maintenance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="warn">
     <property name="text">
      <string>If you use your computer with the AC adapter attached and only infrequently use battery power, battery deterioration may occur faster if the battery is constantly charged at 100%. Lowering the charge threshold for your batery will help increase its lifespan.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="chooser">
     <item>
      <widget class="QLabel" name="battery_label">
       <property name="text">
        <string>Battery:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="battery_chooser">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #d2d2d2;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
     </property>
     <property name="title">
      <string>Charge mode</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="always">
        <property name="text">
         <string>Always fully charge (default)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="always_ac">
        <property name="text">
         <string>Always fully charge (AC Optimized, start at 96% and stop at 100%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="life">
        <property name="text">
         <string>Optimize for battery life (cycles)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="custom">
        <property name="text">
         <string>Custom charge threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="start_at">
          <property name="text">
           <string>Start charging when below:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="start">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stop_at">
          <property name="text">
           <string>Stop charging at:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="stop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>85</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_bar">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
